crypto/s2n_dhe.c

```diff
@@ -160,18 +160,25 @@ int s2n_pkcs3_to_dh_params(struct s2n_dh_params *dh_params, struct s2n_blob *pkc
 int s2n_dh_p_g_Ys_to_dh_params(struct s2n_dh_params *server_dh_params, struct s2n_blob *p, struct s2n_blob *g,
                                struct s2n_blob *Ys)
 {
+    notnull_check(server_dh_params);
```

Comment:
We could use`ENSURE_REF`/`ENSURE_REF_PTR` macro instead here.

Reply:
Implemented as suggested.